extensions/notebook/src/common/iconHelper.ts

```diff
@@ -23,7 +23,7 @@ export class IconPathHelper {
 			light: IconPathHelper.extensionContext.asAbsolutePath('resources/light/delete.svg')
 		};
 		IconPathHelper.folder = {
-			dark: IconPathHelper.extensionContext.asAbsolutePath('resources/dark/folder_inverse.svg'),
+			dark: IconPathHelper.extensionContext.asAbsolutePath('resources/dark/folder.svg'),
```

Comment:
If these are the same then I'd suggest just having a single svg under `resources` and referencing that for both